src/panels/lovelace/hui-unused-entities.ts

```diff
@@ -69,6 +124,20 @@ export class HuiUnusedEntities extends LitElement {
           padding: 4px;
           flex: 1 auto;
         }
+
+        paper-fab {
+          position: sticky;
+          float: right;
```

Comment:
Don't think this does anyhting ?

Reply:
yes, it seems to be what makes it go to the right, the `right: 16px;` doesn't seem to do anything...